playbooks/pmm2-client_integration.yaml

```diff
@@ -0,0 +1,149 @@
+---
+# This playbook does following:
+#   enables Percona testing repository
+#   installs latest version of PMM2 client
+#   does some tests
+
+- hosts: all
+  become: true
+  become_method: sudo
+  vars:
+    pmm_server_address: "@@PMM_SERVER_IP@@"
```

Comment:
It would probably be better to use environment variable and just read it from the playbook: https://docs.ansible.com/ansible/latest/plugins/lookup/env.html
Check here for example: https://github.com/Percona-QA/package-testing/blob/master/playbooks/common_57.yml#L13-L15